client/layouts/default.vue

```diff
@@ -100,10 +100,7 @@
     </v-app-bar>
     <v-main>
       <div class="pa-s">
-        <nuxt
-          keep-alive
-          :keep-alive-props="{ max: 10, exclude: ['fullscreen-playback'] }"
-        />
+        <nuxt keep-alive :max="10" :exclude="['fullscreen-playback']" />
```

Comment:
The Nuxt docs don't mention this being a thing. Do you have a source?

Reply:
Found them referenced in the [Vue](https://vuejs.org/v2/api/#keep-alive) documentation and they looked cleaner to me.

Reply:
Seems like there's some level of abstraction in Nuxt, so it expects the ones we were using previously: https://github.com/nuxt/nuxt.js/blob/dev/packages/vue-app/template/components/nuxt.js#L27-L31

Reply:
This seems a good tradeoff between clean/functional:

![image](https://user-images.githubusercontent.com/10274099/117272263-523e3e80-ae5b-11eb-9d4a-adce21b05057.png)

Reply:
@MrTimscampi doesn't like them in data and I don't think it makes sense in props :p so I'll probably just revert this part.

Reply:
It's not really that I don't like them in the data, it's that data adds a whole layer of reactivity that isn't needed :)

Reply:
That's why I propose setting them up in props, as they don't have the reactivity part.

Reply:
Pretty sure this has already been handled in another pull request so I'll close this one.